src/main.cpp

```diff
@@ -1233,6 +1155,111 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
     return res;
 }
 
+// TODO: move to policy/
+bool HasConflicts(CTxMemPool& pool, const CTransaction& tx, set<uint256>& setConflicts) {
```

Comment:
HasConflicts() is a bit of a weird name for this, given it may return false, yet also set a bunch of conflicts in setConflicts.

Maybe call it "HasReplacableConflicts()"?